L/libRDKit/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+using BinaryBuilder, Pkg
+
+julia_version = v"1.6.0"
+
+name = "libRDKit"
```

Comment:
The name of the project is RDKit
```suggestion
name = "RDKit"
```
Note that you'll need to rename the directory accordingly